frontend_app_gallery/src/container/Gallery.jsx

```diff
@@ -656,7 +740,7 @@ export class Gallery extends React.Component {
                 onClose={this.handleCloseDeleteFilePopup}
               >
                 <div className='gallery__delete__file__popup__body'>
-                  <div className='gallery__delete__file__popup__body__msg'>{props.t('Are you sure ?')}</div>
+                  <div className='gallery__delete__file__popup__body__msg'>{props.t('Are you sure?')}</div>
```

Comment:
if you change a translation key, don't forget to run the `build-transaltion` script

Reply:
I messed you commiting the json files, I'll commit them. Thanks.

Reply:
:+1:
I resolve